bundles/org.eclipse.equinox.log.stream/src/org/eclipse/equinox/internal/log/stream/LogStreamManager.java

```diff
@@ -54,7 +57,9 @@ public void start(BundleContext bc) throws Exception {
 
 	/*
 	 * (non-Javadoc)
-	 * @see org.osgi.framework.BundleActivator#stop(org.osgi.framework.BundleContext)
+	 * 
```

Comment:
The (non-Javadoc) comments pre-dates the @Override so (if desired) these could also be deleted in the future. I did this for most of the non-javadoc in platform a long time ago.

Not related to this change, but just wanted to point out this opportunity.

Reply:
I know, it's on my todo list, but I though this was big enough change as it is ;-)

Reply:
> I know, it's on my todo list, but I though this was big enough change as it is ;-)

Cool, thanks for considering this. I still find this non-javadoc annoying... :-)